metricbeat-tests/go.mod

```diff
@@ -13,3 +13,5 @@ require (
 )
 
 replace github.com/testcontainers/testcontainers-go v0.0.8 => github.com/mdelapenya/testcontainers-go v0.0.9-compose
+
+replace github.com/elastic/metricbeat-tests-poc/cli v0.1.0-rc6 => ../cli
```

Comment:
:heart: thanks!

Reply:
I did not do it before because I expected that path to be absolute. I realised it works when you screenshared the other day :)